jumptrading/aminal
# Right place for config

## Description

Now Aminal uses cross-platform means to find correct place for the configuration file.

Fixes #9 

## Type of change

Please delete options that are not relevant.


## How Has This Been Tested?

- Open console and run `type %USERPROFILE%\.config\aminal\config.toml`. The system must report that there is no such a file. Delete or rename the file if it exists.
- Run Aminal and then close it
- Run `type %USERPROFILE%\.config\aminal\config.toml`. It should show the default configuration just created by Aminal.

**Test Configuration**:

OS:` Windows 10`
Go: `go1.11.2 windows/amd64`
